tools/appveyor.psm1

```diff
@@ -311,6 +311,12 @@ function Invoke-AppVeyorTest
         Write-Host -Foreground Green 'Running all CoreCLR tests..'
     }
 
+    # Remove telemetry semaphore file in CI
+    $TelemetrySemaphoreFilePath = join-path $env:CoreOutput DELETE_ME_TO_DISABLE_CONSOLEHOST_TELEMETRY
```

Comment:
do you mean `join-path` should be `Join-Path`? Shall I fix all of these in this file?